data/blog/ninth-wave-of-bitcoin-grants.mdx

```diff
@@ -0,0 +1,134 @@
+fork incentivized by the Bitcoin Lightning Network. By enabling users to host
+relays and earn sats for sharing bandwidth, El Tor addresses existing
+limitations of the Tor network while supporting anonymous communication. The
+project enhances censorship resistance and facilitates private interactions for
+services like ecash mints, nostr relays, and Bitcoin node communication.
+
+El Tor plans to expand the network's scale and capabilities by onboarding more
+relays globally, and leveraging financial incentives to drive adoption. This
+effort will support greater bandwidth, reliability, and anonymity for users, and
+aims to provide a sustainable model for future development. The project will continue
+refining its codebase and exploring integrations that bring the vision of
+private, decentralized communication closer to reality.
+
+Repository: [eltordev/eltor](https://bitbucket.org/eltordev/eltor)  
+License: 3-Clause BSD
+
+### Nutstash
+
+[Nutstash](https://www.nutstash.app/) is a multi-platform Cashu ecash wallet
+with integrated Lightning and nostr functionality. It enables secure and private
+bitcoin transactions through features like full wallet encryption, mint
+discovery, nostr-wallet-connect, and recovery from seed. Nutstash simplifies
+ecash payments and provides a privacy-focused solution for making bitcoin
+transactions more accessible and practical for everyday use.
+
+Nutstash intends to implement additional features such as offline token
+verification and Cashu payment requests, aligning with recent protocol updates.
+The project will prioritize bug fixes, integrate blinded authentication for
+community mints, and explore on-chain mint functionality using BOLT 12. These
+enhancements aim to make Nutstash a comprehensive and reliable tool for ecash
+transactions in the Bitcoin ecosystem.
+
+Repository: [gandlafbtc/nutstash-wallet](https://github.com/gandlafbtc/nutstash-wallet)  
+License: GPL-3.0
+
+### rust-bitcoin
+
+`rust-bitcoin` is a low-level Bitcoin library written in Rust that allows
+applications to interact with the Bitcoin network. It serves as a foundational
+tool for projects like Bitcoin Dev Kit (BDK) and Lightning Dev Kit (LDK),
+offering reliability and correctness essential for developers building on these
+frameworks. The library also supports language bindings, enabling developers
+from various ecosystems to leverage its capabilities.
+
+The project's focus is on stabilizing core components and releasing version 1.0
+of key crates such as `hex-conservative`, `bitcoin_hashes`, `bitcoin_io`,
+`bitcoin_units`, and `bitcoin_primitives`. These efforts aim to provide stable
+APIs for projects like [BDK](https://github.com/bitcoindevkit#hey-there-) and
+[LDK](https://github.com/lightningdevkit/.github/blob/main/profile/README.md#hey-there-),
+empowering developers to confidently build on a collaborative, clean platform.
+
+Repository: [jamillambert/rust-bitcoin](https://github.com/jamillambert/rust-bitcoin)  
```

Comment:
Could be missing something, but why is the link not to https://github.com/rust-bitcoin/rust-bitcoin?